docs/catalog/web/enable-text-compression.md

```diff
@@ -0,0 +1,45 @@
+---
+version: 1.0
+submitted_by: dubrie
+published_date: TBD
+category: web
+tags: 
+ - text
+ - compression
+ - size:small
```

Comment:
please add `role:web-developer` to the list of tags

Reply:
Done.